packages/Popover/src/Popover.tsx

```diff
@@ -59,7 +59,7 @@ export enum Justify {
   End = 'end',
 }
 
-export interface PopoverProps {
+interface PopoverProps {
```

Comment:
Unless I'm missing something, we need this to be exported.

From `Menu.tsx`:
```js
import Popover, { Align, Justify, PopoverProps } from '@leafygreen-ui/popover';
```

Reply:
this must have been removed by me at some point, was here at one point ??

Reply:
It looks like it was there on master.